### PR TITLE
doc: Add translation for Simplified Chinese(zh-CN)

### PR DESCRIPTION
**Pull request by RiverTwilight**
## Summary/ Motivation (TLDR;)

Translation for zh-CN.

Use the dropdown menu on the top to switch languages.

image

As I didn't find how to customize the navbar for each page, the navbar items have not to be translated.

### CODE REVIEW

File: website/docs/zh/guide/vue-test-utils.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# @vue/test-utils
+
+See more at <https://github.com/nvh95/vitest-preview/tree/main/examples/vue-test-utils>
```
Comment (jsun969) [inline website/docs/zh/guide/vue-test-utils.md:3]: Here is a missing translation

File: website/docs/zh/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: home
+
+title: Vitest Preview
+titleTemplate: Visual Debugging Experience for Vitest
+
+hero:
+  name: Vitest Preview
+  text: 可视化 Vitest 调试体验
+  tagline: 调试 Vitest 测试，如此简单。 🧪🖼⚡️
```
Comment (jsun969) [inline website/docs/zh/index.md:10]:
```suggestion
  tagline: 轻松地调试 Vitest 测试。 🧪🖼⚡️
```

File: website/docs/zh/guide/vue-test-utils.md (hunk 1 above)
Comment (jsun969) [inline website/docs/zh/guide/vue-test-utils.md:3]: Here is a missing translation

File: website/docs/zh/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      link: /zh/guide/getting-started
+    - theme: alt
+      text: GitHub
+      link: https://github.com/nvh95/vitest-preview
+    - theme: brand
+      text: 体验 Demo
+      link: http://bit.ly/try-vitest-preview
+
+features:
+  - title: '体验优先'
```
Comment (jsun969) [inline website/docs/zh/index.md:23]:
```suggestion
  - title: 'DX驱动'
```

File: website/docs/zh/index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    details: "Vitest Preview 的使命是：让前端开发者的生活更轻松。我们始终秉持“开发体验优先”的理念。"
+  - title: '快速'
+    details: Vitest Preview 建立在 Vite 之上，速度相当快——只需几毫秒即可在浏览器中响应更改。
+  - title: 无缝衔接 Vitest
+    details: 只需少量配置，即可在浏览器中可视化你的 Vitest 测试
+  - title: 高质量应用
+    details: Vitest Preview 能帮助你更快更轻松地建立测试。 你的应用也会有更好的可靠性和拓展性。
```
Comment (jsun969) [inline website/docs/zh/index.md:30]:
```suggestion
    details: Vitest Preview 能帮助你更快更轻松地建立测试。你的应用也会有更好的可靠性和拓展性。
```

File: website/docs/guide/getting-started.zh-CN.md [hunk 5]
```diff
@@ -0,0 +1 @@
+dfasf
```
Comment (jsun969) [inline website/docs/guide/getting-started.zh-CN.md:1]: ???

File: website/docs/.vitepress/config.ts [hunk 6]
```diff
@@ -111,3 +141,41 @@ function sidebarGuide() {
     },
   ];
 }
+
+function sidebarGuide_zhCN() {
```
Comment (jsun969) [inline website/docs/.vitepress/config.ts:145]:
```suggestion
function sidebarGuideCN() {
```

File: website/docs/.vitepress/config.ts [hunk 7]
```diff
@@ -22,6 +22,7 @@ export default defineConfig({
     logo: 'logo.svg',
     sidebar: {
       '/guide/': sidebarGuide(),
+      '/zh/guide/': sidebarGuide_zhCN()
```
Comment (jsun969) [inline website/docs/.vitepress/config.ts:25]:
```suggestion
      '/cn/guide/': sidebarGuideCN()
```

File: website/docs/.vitepress/config.ts [hunk 8]
```diff
@@ -38,13 +39,29 @@ export default defineConfig({
...
+
+  locales:{
+     // The key is the path for the locale to be nested under.
+    // As a special case, the default locale can use '/' as its path.
+    '/': {
+      lang: 'en-US',
+      title: 'Vitest Preview',
+      description: 'Visual Debugging Experience for Vitest 🧪🖼⚡️',
+    },
+    '/zh/': {
```
Comment (jsun969) [inline website/docs/.vitepress/config.ts:59]:
```suggestion
    '/cn/': {
```

**Review by jsun969 — CHANGES_REQUESTED** [review]
Is `/.vitepress/cache/deps` a runtime folder? I assume it should be removed.

**Comment by jsun969** [comment]
Use pnpm instead of yarn

File: website/docs/guide/svelte-testing-library.md [hunk 9]
```diff
@@ -1,3 +1,3 @@
 # Svelte Testing Library
 
-See more at <https://github.com/nvh95/vitest-preview/tree/main/examples/svelte-testing-library>
+参见 <https://github.com/nvh95/vitest-preview/tree/main/examples/svelte-testing-library>
```
Comment (jsun969) [inline website/docs/guide/svelte-testing-library.md:3]: This is the origin doc

**Review by jsun969 — CHANGES_REQUESTED** [review]

**Comment by RiverTwilight** [comment]
All addressed. Thanks! @jsun969

**Review by jsun969 — APPROVED** [review]

**Review by nvh95 — APPROVED** [review]
LGTM!

**Comment by nvh95** [comment]
@RiverTwilight Thank you very much for your time invested in `vitest-preview` 🥰
@jsun969 Thanks for your help in reviewing this PR.

**Comment by nvh95** [comment]
@all-contributors please add @RiverTwilight for translation

**Comment by nvh95** [comment]
Hi @RiverTwilight @jsun969. I just found out that the merge commit hasn't been deployed to production. I just cleared cache and manually deploy the Chinese version of the docs. You can check it out at https://www.vitest-preview.com/zh/

Again. Thank you for your help 🎉.

**Pull request merged**